content/references/rippled-api/api-conventions/error-formatting.md

```diff
@@ -99,14 +101,15 @@ For other errors that returned with HTTP status code 200 OK, the responses are f
 
 All methods can potentially return any of the following values for the `error` code:
 
-* `unknownCmd` - The request does not contain a [command](rippled-api.html) that the `rippled` server recognizes.
+* `amendmentBlocked` - The server is [amendment blocked](amendments.html#amendment-blocked) and needs to be updated to the latest version to stay synced with the XRP Ledger network.
+- `invalid_API_version` - The server does not support the [API version number](request-formatting.html#api-versioning) from the request.
```

Comment:
It's a little odd that this seems to be the only one with underscores

Reply:
Yep, it's an inconsistency in the code.